db/sc_config.conf

```diff
@@ -3729,3 +3729,8 @@ SC_ALMIGHTY: {
 	}
 	Icon: "SI_ALMIGHTY"
 }
+SC_SKE: {
+	Visible: true
```

Comment:
I not sure, probably without ``Visible: true`` it can works too?

Reply:
ah yeah. removed already :D